tests/FINDOLOGIC/Tests/DataElementsTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace FINDOLOGIC\Tests;
+
+use FINDOLOGIC\Export\Data\Attribute;
+use FINDOLOGIC\Export\Data\Bonus;
+use FINDOLOGIC\Export\Data\DateAdded;
+use FINDOLOGIC\Export\Data\Description;
+use FINDOLOGIC\Export\Data\Keyword;
+use FINDOLOGIC\Export\Data\Name;
+use FINDOLOGIC\Export\Data\Ordernumber;
+use FINDOLOGIC\Export\Data\Price;
+use FINDOLOGIC\Export\Data\Property;
+use FINDOLOGIC\Export\Data\SalesFrequency;
+use FINDOLOGIC\Export\Data\Sort;
+use FINDOLOGIC\Export\Data\Summary;
+use FINDOLOGIC\Export\Data\Url;
+use FINDOLOGIC\Export\Helpers\EmptyValueNotAllowedException;
+use PHPUnit\Framework\TestCase;
+
+class DataElementsTest extends TestCase
+{
+    public function multiValueItemProvider()
```

Comment:
Add a doc block describing the structure that each data set has.

Reply:
Done

Reply:
What I meant was listing a description of which values is being provided. In this case, something like element value, element class, and whether that value for this particular class should cause an exception. In other words: What's the structure of this mysterious array?

Reply:
Done